insights/parsers/pcs_config.py

```diff
@@ -78,6 +80,7 @@ class PCSConfig(CommandParser):
     """Class to process the output of ``pcs config`` command.
 
     Attributes:
+        cluster_props_dict (dict): A dictionary containing all cluster properties key, value
```

Comment:
Would you think the name `cluster_ properties` (used in #1796) better than this one?